ptp/multipagepdf.py

```diff
@@ -20,7 +20,8 @@ def get_metadata(mets):
             'Title': title,
             'Keywords': publisher.text+" (Publisher)" if publisher is not None else ""}
 
-def read_from_mets(metsfile, filegrp, outputfile, pagelabel='pageId'):
+def read_from_mets(metsfile, filegrp, outputfile, pagelabel='pageId', overwrite=False):
```

Comment:
BTW the keyword args will always be treated as positional args (`*sys.argv[1:]), so I suggest to remove the default values - they will never be used.